### PR TITLE
SUBMARINE-454. Implement frontend of team page in workbench with Angular

### PR DESCRIPTION
**Pull request by kobe860219**
### What is this PR for?
Implement frontend of team page in workbench with Angular

### What type of PR is it?
Feature

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-454

### How should this be tested?
https://travis-ci.org/github/kobe860219/submarine

### Screenshots (if appropriate)
teamDemo

https://imgur.com/84kc6WV

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by kobe860219** [comment]
1.Presenting functions with simulated data at this stage.
2.Currently mainly presents the permission settings of the members of the team.
3.Functions will be cut into small components in the future and continue developing.

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/workspace/team/team.component.ts [hunk 1]
```diff
@@ -18,17 +18,152 @@
...
+  startAddUser(): void {
+    this.isVisible = true;
+  }
+
+  // Add Success
+  createNotification(type: string): void {
+    this.notification.create(
+      type,
+      'Add Successful!',
+      'Make sure thar user check invitation!'
```
Comment (pingsutw) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/workspace/team/team.component.ts:137]:
```suggestion
      'Make sure that user check invitation!'
```

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/workspace/team/team.component.html [hunk 2]
```diff
@@ -17,7 +17,149 @@
...
+<nz-modal
+  [(nzVisible)]="isVisible"
+    nzTitle="Invite new member"
+    (nzOnCancel)="handleCancel()"
+    (nzOnOk)="handleOk()"
+    [nzOkLoading]="isOkLoading"
+>
+  <p>Use E-mail to add member!</p>
+  
+</nz-modal>
```
Comment (jasoonn) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/workspace/team/team.component.html:165]: Should be added newline at the end of the file.

**Comment by jasoonn** [comment]
Thanks for your contribution, @kobe860219
It looks great, and I have some suggestion.
1. Add some basic E2E unit tests: you can refer to this [this](https://github.com/apache/submarine/blob/master/submarine-test/test-e2e/src/test/java/org/apache/submarine/integration/workspaceIT.java). Maybe add a function in the file for testing the team page.
2. There should be newline at the end of all the files.

**Comment by kobe860219** [comment]
Fix the problem in comment and add teamIT.java e2e test file.
teamITDemo

**Comment by kobe860219** [comment]
@jasoonn  @liuxunorg  I have updated done. Please help me preview this PR, thanks a lot.

**Comment by jasoonn** [comment]
@kobe860219 Thank for your contribution. It looks good to me :))

**Comment by liuxunorg** [comment]
Will merge if no more comments.

**Pull request closed without merging**